### PR TITLE
feat(Pipelines): improve pipeline messages UX

### PR DESCRIPTION
**Pull request by cheikhgwane**
Improve pipeline messages UX

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Render url in run message as link 
- Improve multi-line display

## How/what to test

A few words about what needs to be tested, along with specific testing instructions if needed.

## Screenshots / screencast

image

### CODE REVIEW

**Comment by cheikhgwane** [comment]
For the modification on jest.setup.js see issue https://github.com/remarkjs/react-markdown/issues/635. @pvanliefland

File: src/core/components/MarkdownViewer/MarkdownViewer.tsx [hunk 1]
```diff
@@ -31,6 +32,7 @@ const MarkdownViewer = (props: MarkdownViewerProps) => {
     <ReactMarkdown
       className={clsx("prose prose-headings:font-medium", maxWidth, className)}
       components={components}
+      remarkPlugins={[remarkGfm]}
```
Comment (pvanliefland) [inline src/core/components/MarkdownViewer/MarkdownViewer.tsx:35]: Why do we need `remarkGfm` here?

File: src/pipelines/features/RunMessages/RunMessages.tsx [hunk 2]
```diff
@@ -48,11 +48,22 @@ const RunMessages = (props: RunMessagesProps) => {
...
+        accessor="priority"
+        label={t("Priority")}
+        className="max-w-[50ch] py-3"
+      >
         {(value) => <Badge className={getBadgeClassName(value)}>{value}</Badge>}
       </BaseColumn>
-      <TextColumn width={400} accessor="message" label={t("Message")} />
+      <BaseColumn accessor="message" label={t("Message")} width={400}>
+        {(value) => (
+          <MarkdownViewer className="w-max-[30ch] text-sm overflow-y-auto whitespace-pre-line break-words">
```
Comment (pvanliefland) [inline src/pipelines/features/RunMessages/RunMessages.tsx:62]: Could we forward `allowedElements={['a']}` to the underlying component? We don't want to support other tags at this stage.

https://github.com/remarkjs/react-markdown#options

**Review by pvanliefland — CHANGES_REQUESTED** [review]
Hey @cheikhgwane , good work, a couple of questions

File: src/core/components/MarkdownViewer/MarkdownViewer.tsx (hunk 1 above)
Comment (cheikhgwane) [reply src/core/components/MarkdownViewer/MarkdownViewer.tsx:35]: To render url as <a></a> tag.

File: src/pipelines/features/RunMessages/RunMessages.tsx (hunk 2 above)
Comment (cheikhgwane) [reply src/pipelines/features/RunMessages/RunMessages.tsx:62]: If we do this it'll prevent all the others tag from being rendered (in our case message will be empty)

Comment (cheikhgwane) [reply src/pipelines/features/RunMessages/RunMessages.tsx]: In that case we'll also add `allowedElements={['p','a']}`

**Review by pvanliefland — APPROVED** [review]

**Comment by qgerome** [comment]
AFAIK the module "remark-gfm" was not in package.json

**Comment by pvanliefland** [comment]
@qgerome yeah it's an indirect dependency from another module I guess. Shall we add it in `package.json`?

**Comment by qgerome** [comment]
Yes. I already fixed it.

**Pull request merged**